### DISCUSSION TITLE
EmrStepSensor does not support reschedule mode for Airflow version 2.5.1

### DISCUSSION THREAD

**Discussion by sonalprsd** [apache/airflow#32480] [Q&A]
The EMRStepSensor is now decorated with poke_mode_only which only allows the mode to be set to poke. 
1. Which airflow version is reschedule mode supported for EMRStepSensor?
2. What are the challenges due to which it is currently not supported?

**Comment by potiuk** [comment]
Have you seen changelog ? https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/index.html I believe it has been removed already in latest version and you can see history of chnages there.

**Comment by potiuk** [comment]
Basically best to look at the changelog and history of changes in git if you have such detailed questions - this will give you much more accurate answers than anyone's memory. Airflow has few thousands of operators, and the history of changes is likely the only place you can get authoritative answers.
